### DISCUSSION TITLE
I am importing airtable data to nocodb [Feb 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5087] [Discord Threads]
> 🙋‍♂️ **Akashkc**  
## I am importing airtable data to nocodb through API and it has 20k rows and 9 to 10 sheets and taking 6 to 7 hours but I am unable to import Images proparly as you can see in image  and redirecting to website. Is there any way to import image successfully from airtable? (I have hosted nocodb in gcb where image is not importing in correct format)  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1061250285139415091/image.png)  
![image1](https://cdn.discordapp.com/attachments/661905455894888493/1061250285592391780/image.png)  
![image2](https://cdn.discordapp.com/attachments/661905455894888493/1061250285906972786/image.png)  
![image3](https://cdn.discordapp.com/attachments/661905455894888493/1061250286347366470/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **Acedia**  
#### Hi <@1060510421930475530>, for attachment import issue please use NC_BASEURL_INTERNAL environment variable to provide external URL you are reaching to your docker (before running AT Import job)  
#  
> 🙋‍♂️ **Acedia**  
#### You can follow further updates on this topic from https://github.com/nocodb/nocodb/issues/4185 issue  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1061250286670336090)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
